### DISCUSSION TITLE
The update of replit ide

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#163] [General]
I love how you can customize the template that the html generates, I think I will start using the repl, since now it is much better, and more with the asset library

**Comment by slmjkdbtl** [comment]
Autocomplete is coming next week which'll also make stuff a lot easier. And the sprites asset library is going to get revamped to all hand-drawn assets like in [here](https://github.com/replit/kaboom/tree/master/sprites)

**Comment by slmjkdbtl** [comment]
Please leave any feedbacks here 🙏  I've heard sometimes it's slowing down the editing (typing).

**Reply by lajbel** [reply]
now Kaboom focuses more on an animated art style (like flash)?

**Reply by slmjkdbtl** [reply]
Not really yet, moving to a more vector graphics style now (solid color + thick outline) with no animations baked in yet. Planning to make a vector graphics engine in kaboom that enables you to programmatically create some animations, kinda like p5.js but game assets oriented.

But you can always completely get rid of the defaults and bring your own style.

**Reply by lajbel** [reply]
I understand, it reminds me of stencyl, very cool

**Comment by lajbel** [comment]
I would also recommend a section to load components and plugins, and be used in the code section

**Comment by loretod** [comment]
I'm wondering if there is way to add cdn projects in the Replit Kaboom editor? I would like to use the Handsfree.js to make fine motor games. The two integrate nicely if I use a standard HTML Replit but loading the CDN in the index.html file on a Kaboom Replit. The CDN is removed once I run the project. Not sure if this makes any sense but it would be a really nice feature since the Kaboom Replit is lovely so far.

**Reply by lajbel** [reply]
Edit your template and add the cdn's
![image](https://user-images.githubusercontent.com/71136486/132022051-1939979c-c9fd-4a81-9682-2f61dd765306.png)
![image](https://user-images.githubusercontent.com/71136486/132022192-c228bcd0-cd6f-4642-8431-6507ea83772f.png)

**Reply by slmjkdbtl** [reply]
@loretod I think you're using the old kaboom (beta), a new version will be released next week where you can customize the HTML and everything!

**Reply by loretod** [reply]
Woot!! That's great news! Yes, I'm using the beta. Thanks for the quick reply. I will check back again next week!

**Reply by slmjkdbtl** [reply]
I'll also message you here when it happened :D

**Reply by loretod** [reply]
Hurray! I just checked back today and opened up the non-legacy Kaboom. I was able to integrate handsfree.js on the tutorial project (very nice by the way): [https://testkaboom.loretod.repl.co/](https://testkaboom.loretod.repl.co/). Detecting your left hand jumps the sprite. You have to run it in a new tab and not the editor. For some reason even though I give camera permissions it doesn't work in the editor. Super excited to build! Thanks for the hard work!

**Reply by slmjkdbtl** [reply]
Ah it was released 2 days ago, sorry forgot to notify you here! Glad you liked it :D Is the webview in replit editor unable to use the camera? Not sure, maybe they have some more security constraint added to that webview, I'll check.
